packages/emoji-mart/src/components/Picker/Picker.tsx

```diff
@@ -1082,6 +1082,7 @@ export default class Picker extends Component {
           ref={this.refs.skinToneButton}
           class="skin-tone-button flex flex-auto flex-center flex-middle"
           aria-selected={this.state.showSkins ? 'true' : 'false'}
+          aria-expanded={this.state.showSkins ? 'true' : 'false'}
```

Comment:
Just for context, this is needed because it triggers an overlay. Adding aria-expanded makes this button a `menu-button`.

Not entirely sure this is 100% correct, as the list with the different skin-tones is not an actual menu.. it's a radiogroup with radio inputs. Having the list with options as radiogroup makes sense, we believe that's correct. But, the a11y team was complaining that the button needs to have role menu-button, which is why we added the aria-expanded. 

I just realised that it also needs the following 2 attributes:
```
aria-controls="skin-tone-options"
aria-haspopup="true"
```

SR now announces:
<img width="576" alt="image" src="https://github.com/user-attachments/assets/e6e80f0a-5ba6-4561-a950-37c1d8bba5ae">

Instead of:
<img width="568" alt="image" src="https://github.com/user-attachments/assets/0f60fb01-8a3a-4d35-8592-ab22612e5bf2">

Hope this makes sense?

Reply:
I'm confident this is fine, as the SR never mentions of a menu.